android/java/org/chromium/chrome/browser/app/domain/PortfolioModel.java

```diff
@@ -131,7 +131,7 @@ public NftDataModel(
         }
     }
 
-    public class Erc721MetaData implements Serializable {
```

Comment:
Better to make these two classes static. 
Without it, `Erc721MetaData` was holding a reference to the outer class, and the serialization was failing.